docs/sources/datasources/influxdb/template-variables/index.md

```diff
@@ -87,7 +87,7 @@ When you enable the **Multi-value** or **Include all value** options, Grafana co
 
 ### Templated dashboard example
 
-To view an example templated dashboard, refer to [InfluxDB Templated Dashboard](https://play.grafana.org/dashboard/db/influxdb-templated).
+To view an example templated dashboard, refer to [InfluxDB Templated Dashboard](https://play.grafana.org/d/f62a0410-5abb-4dd8-9dfc-caddfc3e2ffd/eccb2445-b0a2-5e83-8e0f-6d5ea53ad575).
```

Comment:
Since this dashboard title is really long, i've just taken this out of title case so it's a more generic reference.
```suggestion
To view an example templated dashboard, refer to this [InfluxDB templated dashboard](https://play.grafana.org/d/f62a0410-5abb-4dd8-9dfc-caddfc3e2ffd/eccb2445-b0a2-5e83-8e0f-6d5ea53ad575).
```